packages/manager/cypress/support/intercepts/configs.ts

```diff
@@ -72,3 +74,64 @@ export const interceptDeleteLinodeConfig = (
     apiMatcher(`linode/instances/${linodeId}/configs/${configId}`)
   );
 };
+
+/**
+ * Mocks DELETE request to delete an interface of linode config.
+ *
+ * @param linodeId - ID of Linode for intercepted request.
+ * @param configId - ID of Linode config for intercepted request.
+ * @param interfaceId - ID of Interface in the Linode config.
+ *
+ * @returns Cypress chainable.
+ */
+export const mockDeleteLinodeConfignInterface = (
```

Comment:
```suggestion
export const mockDeleteLinodeConfigInterface = (
```
I think this is a small typo. 🔍